supertokens_python/recipe/session/asyncio/__init__.py

```diff
@@ -102,7 +107,6 @@ async def create_new_session_without_request_response(
     final_access_token_payload = {**access_token_payload, "iss": issuer}
 
     for claim in claims_added_by_other_recipes:
-        # TODO: Pass tenant id
         update = await claim.build(user_id, "pass-tenant-id", user_context)
```

Comment:
pass in tenant id.. you removed the TODO, but did not actually pass in the tenant id. Search the whole code base for `pass-tenant-id`

Reply:
Done